src/videostreamindex/DefaultVideoStreamIndex.ts

```diff
@@ -215,6 +215,14 @@ export default class DefaultVideoStreamIndex implements VideoStreamIndex {
     return this.highestQualityStreamFromEachGroupExcludingSelf(selfAttendeeId).array().length;
   }
 
+  numberOfParticipants(): number {
+    if (!!this.currentIndex.numParticipants) {
```

Comment:
when will this be undefined ?

Reply:
This comes from the INDEX message, and is an optional field of the protobuf message.   This would handle the case where that optional field is not in the protobuf message